app/views/admin/service_announcements/_menu.html.haml

```diff
@@ -0,0 +1,21 @@
+.ui.fluid.vertical.menu.sticky{ data: { behavior: "sticky", context: "#stickContext", offset: 75 } }
+  .item
+    .header
+      Service Announcements
+
+  = active_link_to admin_service_announcements_path, active: :exclusive, class: "item" do
+    Index
+    - if count
+      .ui.label= count
+
+  = active_link_to new_admin_service_announcement_path, class: "item" do
+    %i.plus.icon
+    New
+
+  -#.item
+    .ui.transparent.icon.input
+      = form_tag admin_service_announcements_path, method: :get do
```

Comment:
hi josh

Reply:
I'm gonna add in searching eventually for these :P